code/iaas/lb-instance/src/main/java/io/cattle/platform/lb/instance/process/LoadBalancerUpdateConfig.java

```diff
@@ -1,6 +1,6 @@
 package io.cattle.platform.lb.instance.process;
 
-import static io.cattle.platform.core.model.tables.PortTable.*;
+import static io.cattle.platform.core.model.tables.PortTable.PORT;
```

Comment:
FYI, Darren has a preference for dong * for any number of static imports.
He had me change `Java > Code Style > Organize Imports > Number of static imports needed...` to 1.
Just letting you know; no need to change this instance.